posts/2024/2024_08_22_inigo_final_report.rst

```diff
@@ -0,0 +1,309 @@
+.. role:: custom-title
+
+.. |main-title| raw:: html
+
+   <span class="gsoc-title">Google Summer of Code Final Work Product</span>
+
+.. |project-abstract-title| raw:: html
+
+   <span class="gsoc-title">Project Abstract</span>
+
+.. |proposed-objectives-title| raw:: html
+
+   <span class="gsoc-title">Proposed Objectives</span>
+
+.. |modified-objectives-title| raw:: html
+
+   <span class="gsoc-title">Modified Objectives (Additional)</span>
+
+.. |objectives-completed-title| raw:: html
+
+   <span class="gsoc-title">Objectives Completed</span>
+
+.. |objectives-progress-title| raw:: html
+
+   <span class="gsoc-title">Objectives in Progress</span>
+
+.. |future-work-title| raw:: html
+
+   <span class="gsoc-title">Future Works</span>
+
+.. |open-source-tasks-title| raw:: html
+
+   <span class="gsoc-title">Open Source Tasks</span>
+
+.. |pull-requests-title| raw:: html
+
+   <span class="gsoc-title">Pull Requests</span>
+
+.. |conclusions-title| raw:: html
+
+   <span class="gsoc-title">Conclusions</span>
+
+.. |timeline-title| raw:: html
+
+   <span class="gsoc-title">Timeline</span>
+
+.. image:: /_static/images/logos/gsoc-logo.png
+   :height: 40
+   :target: https://summerofcode.withgoogle.com/programs/2024/projects/dHajBmW3
+   :class: no-background
+
+.. image:: /_static/images/logos/python-logo.png
+   :height: 40
+   :target: https://summerofcode.withgoogle.com/programs/2024/organizations/python-software-foundation
+   :class: no-background
+
+.. image:: /_static/images/logos/dipy-logo-2.png
+   :height: 30
+   :target: http://dipy.org
+   :class: no-background
+
+.. raw:: html
+
+   <div style="margin-top: 20px;"></div>
+
+|main-title|
```

Comment:
Hi @itellaetxe, 

you did a great job. And your blog is also clear and pretty, I just have some proposals to make it more convenient for readers see below.

Reply:
Thank you for your feedback and your comments Wachiou